apps/costcontrol/test/unit/settings_test.js

```diff
@@ -538,6 +538,44 @@ suite('Settings Test Suite >', function() {
         Settings.initialize();
       });
 
+      test('No corrections with invalid entries', function(done) {
```

Comment:
Hi,
I think it's not necessary add a specific test for the 0.1 value because of 0.1 is considered a valid value. The root cause of the regression is the values '0, 0., 0.0, ., .0' are not allowed.